Makefile

```diff
@@ -68,7 +68,7 @@ local-gadget-%: phony_explicit
 	if [ "$(shell go env GOOS)" = "$${GOOS}" -a "$(shell go env GOARCH)" = "$${GOARCH}" ] ; then \
 		export CGO_ENABLED=1 && \
 		export GO111MODULE=on && \
-		make -C gadget-container ebpf-objects && \
+		make -C gadget-container && \
```

Comment:
We should just delete this line (and possibly replace it with `go generate ./...`?). Since the default target of `gadget-container/Makefile` is `gadget-container-deps`, this change will now build `gadgettracermanager` and other binaries, which does not make sense for local-gadget.

Ditto below line 152 in the `local-gadget-tests` target.

Reply:
Good catch. For the time being I think the right solution is to remove those files, as replacing with `go generate ./...` will cause the ebpf objects to be recompiled again, what I'm not sure is a good idea.